File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/rooftop-solar-project-management-page/_models/rooftop-solar-project-management.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface RooftopSolarProjectManagementModel extends BaseModel {
  rooftopSolarProjectManagementId: string;
  projectName: string; //Tên dự án
  investorName: string; //Chủ đầu tư
  address: string; //Vị trí
  area: number | null; //Diện tích
  surveyPolicy: string; //Chủ trương khảo
  wattage: number | null; //Công suất
  progress: string; //Tiến độ
  district: string;
  operationDay: any;
}

File: sct_source/SourceCode/API_SoCongThuong/Models/RooftopSolarProjectManagementModel.cs
namespace API_SoCongThuong.Models
{
    public class RooftopSolarProjectManagementModel
    {
        /// <summary>
        /// Quản lý dự án điện mặt trời áp mái
        /// </summary>
        public Guid RooftopSolarProjectManagementId { get; set; }
        /// <summary>
        /// Tên dự án
        /// </summary>
        public string ProjectName { get; set; } = null!;
        /// <summary>
        /// Nhà đầu tư
        /// </summary>
        public string InvestorName { get; set; } = null!;
        /// <summary>
        /// Vị trí
        /// </summary>
        public string Address { get; set; } = null!;
        /// <summary>
        /// Diện tích
        /// </summary>
        public decimal Area { get; set; }
        /// <summary>
        /// Chủ trương khảo
        /// </summary>
        public string SurveyPolicy { get; set; } = null!;
        /// <summary>
        /// Công suất
        /// </summary>
        public decimal Wattage { get; set; }
        /// <summary>
        /// Tiến độ
        /// </summary>
        public string Progress { get; set; } = null!;
        public bool IsDel { get; set; }
        public DateTime CreateTime { get; set; }
        public Guid CreateUserId { get; set; }
        public DateTime? UpdateTime { get; set; }
        public Guid? UpdateUserId { get; set; }
        public Guid? District { get; set; } = Guid.Empty;
        public string DistrictName { get; set; } = "";
        public DateTime? OperationDay { get; set; }
    }
}